huaweicloud/services/acceptance/bms/resource_huaweicloud_bms_instance_test.go

```diff
@@ -163,10 +167,18 @@ resource "huaweicloud_bms_instance" "test" {
     key = "value"
   }
 
+  system_disk_type = "GPSSD"
+  system_disk_size = 150
+
+  agency_name = "aom_admin_trust"
+
+  metadata = {
+    __bms_support_evs = "true"
+  }
+
   charging_mode = "prePaid"
   period_unit   = "month"
   period        = "1"
-  auto_renew    = "false"
```

Comment:
Please explain the reason why removing `auto_renew`.
This kind of change needs to be explained in the PR.